package.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "floeproject.org",
+    "version": "1.0.0",
+    "description": "FLOE Project Website",
+    "scripts": {
```

Comment:
Add a `clean` script that removes node modules and dist.

For an example, please see: https://github.com/inclusive-design/idrc/blob/01d005b0483e5f7654c291bfbf508a5a2de46248/package.json#L12

Reply:
fixed